remix-tests/src/compiler.js

```diff
@@ -59,7 +74,7 @@ function compileFileOrFiles (filename, isDirectory, opts, cb) {
       compiler.event.register('compilationFinished', this, function (success, data, source) {
         next(null, data)
       })
-      compiler.compile(sources, filepath)
+      compiler.compile(sources, false)
```

Comment:
why this change? i think the second param is still a target path.

Reply:
should we rollback this change ?

Reply:
Not sure. Ok I will take a look.

Reply:
Yeah we should use `filepath`. Added back on commit 8237113.